app/assets/stylesheets/ama_layout/webfonts/ss-symbolicons-block.scss

```diff
@@ -0,0 +1,50 @@
+@charset "UTF-8";
```

Comment:
@kathleenAMA can you move this under `vendor/assets/stylesheets/.keep`:

`git mv app/assets/stylesheets/ama_layout/webfonts/ss-symbolicons-block.scss vendor/assets/stylesheets/`